clang/lib/Serialization/ASTReader.cpp

```diff
@@ -2419,6 +2422,9 @@ namespace {
       // declarations it needs.
       ++NumIdentifierLookupHits;
       Found = *Pos;
+      if (Trait.hasMoreInformationInDependencies())
+        // Look for the identifier in extra modules as they contain more info.
+        return false;
```

Comment:
Nit: I'd wrap this in braces since it spans two lines.

Reply:
Done.